primitives.js

```diff
@@ -0,0 +1,2 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+module.exports = require('./lib/primitives')
```

Comment:
Ah yes sorry I hadn't noticed those being added. I vaguely prefer `/native.js` rather than `/native/index.js` since they're just aliases into `/lib`, but I don't know where the typescript definitions would live then (`/typings` perhaps?).

Anyway, we can figure that out later. For now, better to only have one place for everything, so you can delete these two files.